src/sparql/BaseSparqlParser.ts

```diff
@@ -353,13 +372,10 @@ export class BaseSparqlParser extends Parser implements IStardogParser {
 
   Update = this.RULE('Update', () => {
     log('Update');
-    this.SUBRULE(this.Prologue);
-    this.OPTION(() => {
-      this.SUBRULE(this.Update1);
-      this.OPTION1(() => {
-        this.CONSUME(sparqlTokenMap.Semicolon);
-        this.SUBRULE(this.Update);
-      });
+    this.SUBRULE(this.Update1);
+    this.OPTION1(() => {
```

Comment:
Can this just be `this.OPTION`?

Reply:
Doesn't really make a difference, but yeah, done.